PyDSS/dssInstance.py

```diff
@@ -297,9 +297,9 @@ def _GetRelaventObjectDict(self, key):
 
     def RunStep(self, step, updateObjects=None):
         # updating parameters bebore simulation run
-        time_step_has_converged = True
-        self._Logger.info(f'PyDSS datetime - {self._dssSolver.GetDateTime()}')
-        self._Logger.info(f'OpenDSS time [h] - {self._dssSolver.GetOpenDSSTime()}')
+        if self._Options["Logging"]["Log time step updates"]:
+            self._Logger.info(f'PyDSS datetime - {self._dssSolver.GetDateTime()}')
```

Comment:
This was causing excessive logging, so I disable it in DISCO.

Reply:
Okay